### PR TITLE
Added support for an additional font

### PR DESCRIPTION
**Pull request by hokei**
This is just using a placeholder from google fonts -- will replace with the URL of OpenDyslexia when we get that up.

### CODE REVIEW

File: src/IFrameNavigator.ts [hunk 1]
```diff
@@ -234,6 +237,7 @@ export default class IFrameNavigator implements Navigator {
     (document as any).msFullscreenEnabled;
 
   public static async create(config: IFrameNavigatorConfig) {
+    console.log("config", config);
```
Comment (kristojorg) [inline src/IFrameNavigator.ts]: Extra console

File: test/IFrameNavigatorTests.ts [hunk 2]
```diff
@@ -2594,7 +2594,7 @@ describe("IFrameNavigator", () => {
       const titlePage = `<?xml version="1.0" encoding="UTF-8" ?>
 <html>
 <head>
-<title>Book Tittle</title>
```
Comment (kristojorg) [inline test/IFrameNavigatorTests.ts:2606]: I prefer tittle

**Review by kristojorg — COMMENTED** [review]
Is this meant to be enabled in the settings panel? I'm not seeing it there...

**Review by kristojorg — APPROVED** [review]
Nevermind, I ran it with the right CPW branch and it worked!

**Comment by hokei** [comment]
> Is this meant to be enabled in the settings panel? I'm not seeing it there...


Yeah, you gotta enable it when you call the reader, and I think it doesn't need to be enabled by default on webpub-viewer, but it /should/ live in webpub viewer since it's something that non-CPW people might want to offer!

File: src/DyslexiaFont.ts [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+  public readonly label = "Dyslexia Friendly";
+
+  public bookElement: HTMLIFrameElement;
+
+  public start(): void {
+    const rootFrame = HTMLUtilities.findRequiredIframeElement(this.bookElement.contentDocument, "html") as HTMLHtmlElement;
+
+    HTMLUtilities.setAttr(rootFrame, "data-viewer-font", "dyslexia-friendly");
+    HTMLUtilities.createStylesheet(rootFrame, "dyslexia-font-stylesheet", 
+    `@font-face {
```
Comment (hokei) [inline src/DyslexiaFont.ts:15]: yes, this is the way to load a new font inside this iframe.   No, I haven't figured out how to do this via a css file.

**Comment by kristojorg** [comment]
Appears to be working on local with the newest beta

**Review by kristojorg — APPROVED** [review]

File: src/DyslexiaFont.ts (hunk 3 above)
Comment (kristojorg) [reply src/DyslexiaFont.ts:15]: lol

**Pull request merged**